src/main.rs

```diff
@@ -94,7 +88,7 @@ struct Opt {
     cache_path: PathBuf,
 
     #[structopt(long, default_value = "10 GB", parse(try_from_str = Self::non_zero_bytes))]
-    cache_size: u64,
+    max_size: u64,
```

Comment:
`max_cache_size`? Who parses `10 GB` to `10 * 1024**3`

Reply:
>  Who parses 10 GB to 10 * 1024**3

The `structopt` library that provides the `#[structopt(...)]` macro